lib/command.js

```diff
@@ -1256,81 +1368,87 @@ Expecting one of '${allowedValues.join("', '")}'`);
    * @api private
    */
 
-  _parseCommand(operands, unknown) {
-    const parsed = this.parseOptions(unknown);
-    this._parseOptionsEnv(); // after cli, so parseArg not called on both cli and env
-    this._parseOptionsImplied();
-    operands = operands.concat(parsed.operands);
-    unknown = parsed.unknown;
-    this.args = operands.concat(unknown);
+  _parseCommand(operands, unknown, async) {
+    this._asyncParsing = async;
```

Comment:
Is the `_asyncParsing` property bring used to track whether in the middle of a phase, rather than "async"?

Reply:
Yes, for motivation see commit message of [94e439d](https://github.com/tj/commander.js/pull/1919/commits/94e439d75d4b7510dc06cbed6e741f235034f236). #1917 does not use `_asyncParsing` anymore, but there would still be a nasty redundancy that I think would not even result in a merge conflict when #1915 is merged, which I do hope will happen eventually. Maybe it is not such a good idea to keep other PRs in mind, though. I am not sure since I have almost zero experience with contributing to open source.

Reply:
> Maybe it is not such a good idea to keep other PRs in mind, though.

In my opinion it is ok to keep them in mind, and helpful to warn about potential merge issues. 

(Even ok in my opinion to suggest it should land after another PR, but then be prepared for a rewrite if that other one gets rejected!)